### DISCUSSION TITLE
Web application security support

### DISCUSSION THREAD

**Discussion by vsecv** [EMResearch/EvoMaster#945] [Q&A] [answered]
Hello,

Can the tool also detect web application security issues?

For example, XSS?

Thanks

**Comment by arcuri82** [comment]
Hi,
 no, currently we do not yet deal with security aspects (it is work in progress).

**Comment by arcuri82** [comment, marked as answer]
as of version 5.2.0, EvoMaster does not different kinds of security attacks, including XSS
